### DISCUSSION TITLE
Bug: go-cqhttp无法扫码登陆

### DISCUSSION THREAD

**Discussion by yulongyu** [koishijs/koishi#1188] [Q&A]
### Describe the bug

使用onebot当qq机器人，提示当前协议不支持二维码登录，请配置账号密码或更换协议。
协议使用的是ws-reverse ，但是我本地自己装的go-cqhttp是能正常扫码登陆的，想问下能不能onebot机器人使用本地的go-cqhttp，而不是插件的go-cqhttp，或者有没有其他方案解决这个扫码登陆问题？

### Steps to reproduce

onebot qq机器人 登陆

### Expected behavior

qq扫码登陆

### Versions

- OS: windows 10 server

- Koishi version:4.14.3

**Comment by LeeJSwit-HCN** [comment]
见gocqhttp的搭建方法。
在onebot插件内更改gocq的地址就行了

似乎现在gocq不再支持扫码登录了（qsign闹的）
扫码登录应该是好几个版本之前的事了

**Comment by yulongyu** [comment]
> 
我刚简单粗暴的把本地的覆盖到插件里面了 好像能登陆了 现在使用画图碰到个其他问题 ai画图也是你在维护么

**Comment by LeeJSwit-HCN** [comment]
> 也

不再维护了，今天最后一天维护我的小机器人。

**Comment by ilharp** [comment]
Koishi 的使用有关问题请前往 [Discussions 版](https://github.com/koishijs/koishi/discussions) 继续讨论，Issue 区只用于反馈 Koishi 的 bug 和功能建议。

需要注意：如果你使用 rryth 插件的话，楼上同学并非 rryth 插件的开发者或关系者。Koishi 论坛仍然有开发者提供免费的 AI 绘图服务，最新的 AI 绘图插件请前往 Koishi 论坛获取。

**Comment by shigma** [comment]
你都已经会使用外置 gocq 了就不需要这个插件了。onebot 可以直接连接外置 gocq 的。
